modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFClaim.testcase

```diff
@@ -93,75 +101,141 @@ definition {
 	@description = "This is a test for LPS-175292. Verify that is possible to export all visible Claims"
 	@priority = 4
 	test CanExportAllVisibleClaims {
-		task ("Given I go to add a new MDF Request with two activities") {
-			PRMUtils.gotoPRM();
+		task ("Given that the Manager user is logged in and goes to MDF listing page") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "manager@liferay.com",
+				userLoginFullName = "manager");
 
-			PRMNavigator.goToNavItem(navItem = "Marketing");
+			PRMNavigator.openSitePage(pageName = "MDF Requests");
+		}
 
-			PRMMarketing.gotoAddNewMDF();
+		task ("And the manager approves the MDF") {
+			PRMMDFRequest.goToMDF(campaignName = "Campaign Name");
 
-			PRMMDFRequest.fillGoalsPage(
-				accountName = "Company Name 1",
-				campaignDescription = "Campaign Description",
-				campaignName = "Campaign Name 2",
-				countryName = "Global",
-				goalsOptions = "Lead generation",
-				targetMarketOptions = "Education,Energy",
-				targetRolesOptions = "Administrator");
+			PRMMDFRequest.changeMDFStatus(statusButton = "Approve");
+		}
 
-			Button.click(button = "Continue");
+		task ("And Login with the Partner user and go to Partner Portal") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "partner@liferay.com",
+				userLoginFullName = "partner");
 
-			PRMMDFRequest.newActivity(
-				activityDescription = "Activity Description",
-				activityName = "Activity Name 1",
-				budgetList = "Sponsorship Fee-2000",
-				followUpStrategy = "test",
-				leadFollowUpList = "Other",
-				leadList = "No",
-				tactic = "Other",
-				targetOfLeads = "test",
-				typeOfActivity = "Miscellaneous Marketing");
+			PRMNavigator.openSitePage(pageName = "MDF Requests");
+		}
 
-			Button.click(button = "Continue");
+		task ("And the partner user makes two claims") {
+			PRMMDFRequest.goToMDF(campaignName = "Campaign Name");
 
-			PRMMDFRequest.newActivity(
-				activityDescription = "Activity Description",
+			PRMMDFClaim.createNewClaim(
+				listLeads = "Document_1.xlsx",
+				reimbursementInvoice = "Document_1.png",
+				thirdInvoice = "Document_1.png");
+
+			PRMMDFClaim.createNewClaim(
 				activityIndex = 1,
-				activityName = "Activity Name 2",
-				budgetList = "Sponsorship Fee-2000",
-				followUpStrategy = "test",
-				leadFollowUpList = "Other",
-				leadList = "No",
-				tactic = "Other",
-				targetOfLeads = "test",
-				typeOfActivity = "Miscellaneous Marketing");
+				listLeads = "Document_1.xlsx",
+				reimbursementInvoice = "Document_1.png",
+				thirdInvoice = "Document_1.png");
+		}
+
+		task ("When Login with the Manager user and go to Partner Portal") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "manager@liferay.com",
+				userLoginFullName = "manager");
+
+			PRMNavigator.openSitePage(pageName = "MDF Requests");
+		}
+
+		task ("And I go to MDF claim listing page") {
```

Comment:
Hi @marinaasant0s, it's a little confuse. I think it should be 'And go to MDF claim listing page'